src/controllers/users.ts

```diff
@@ -0,0 +1,32 @@
+import User from '../models/user';
+import { CreateUserParams } from '../interfaces';
+
+export const getUsers = async () => {
+  const users = await User.findAll();
+  return users;
+};
+
+export const getUserById = async (userId: number) => {
+  const user = await User.findOne({
+    where: { id: userId },
+  });
+  return user;
+};
+
+export const createUser = async (createUserParams: CreateUserParams) => {
+  await User.create(createUserParams);
+};
```

Comment:
Let's use implicit return here

```suggestion
export const createUser = async (createUserParams: CreateUserParams) => 
  await User.create(createUserParams);
```